### DISCUSSION TITLE
newline character  in config file

### DISCUSSION THREAD

**Discussion by bohorok** [polybar/polybar#2965] [General]
Sorry for this question but i searched but could not find.
I've long line in config file. I would like to break the line so that this new one is a continuation of the previous one. I tried: `\n`, `\r`, but not work this.  Could someone suggest what I should use. Thanks in advance for your help.

**Comment by patrick96** [comment]
There is no such thing in polybar. Every line in the config file is its own line with no way to break it up into multiple lines.

There is a plan to add support for this in #2354, but not much work has been done on that.

**Comment by bohorok** [comment]
Great thanks fo very fast answer.
